plugins/processors/converter/converter.go

```diff
@@ -345,6 +363,32 @@ func toFloat(v interface{}) (float64, error) {
 	return internal.ToFloat64(v)
 }
 
+func base64ToFloat32(encoded string) (float32, error) {
```

Comment:
Is it possible to use library functions for this? I see a function that may work for this, `math.Float32frombits()`